### PR TITLE
HTML CSS setup

### PR DESCRIPTION
**Pull request by anuarshaidenov**
## Set up all the HTML and CSS related files
In this pull request I:
- Setup initial folder structure for the styles
- Created markup for the header and hero sections
- Styled header and hero sections
- Setup the linters

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          justify-content: space-between;
+  padding: 0.5rem 0;
+}
+
+.mobile-nav__logo {
+  font-weight: 600;
+}
+
+.section-hero {
+  background-image: url(../../images/mobile/header-illsutration-mobile.svg);
```
Comment (mikethreels) [inline styles/main.css:79]: - [ ] When cloning your repo and opening the webpage the background image cannot be found, In my case, it was solved when you adjust the path of this background image accordingly.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          />
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <section class="section-hero">
+        <div class="container">
+          <h1 class="heading">
+            Hey there. I'm Anuar <br />
+            <span>I'm a software developer</span>
```
Comment (mikethreels) [inline index.html:28]: - [ ] Really great job on the project 👍 according to the project requirements however the subtitle should be using a header tag `hx` (examples: `<h1>`,`<h2>`,`<h3>`, etc...)

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Anuar, 

Wow! great job so far! 🥇 
There are some minor requirements that you need to add to your project to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/main.css (hunk 1 above)
Comment (anuarshaidenov) [reply styles/main.css:79]: @mikethreels It's strange. The path seems correct and works for me. Can you specify how exactly you managed to fix the error?

**Pull request merged**